db/models.py

```diff
@@ -14,3 +14,42 @@ class Actor(models.Model):
 
     def __str__(self) -> str:
         return f"{self.first_name} {self.last_name}"
+
+
+class Movie(models.Model):
+    title = models.CharField(max_length=255)
+    description = models.TextField()
+    actors = models.ManyToManyField(Actor, related_name="actors_set")
+    genres = models.ManyToManyField(Genre, related_name="genres_set")
```

Comment:
I'm not really sure about this, but wouldn't it be better to set `related_name` to a simple plural form? e. g. `actors`, `genres`.